corehq/apps/userreports/tests/utils.py

```diff
@@ -41,12 +41,13 @@ def get_sample_registry_data_source(**kwargs):
 
 
 def bootstrap_pillow(pillow, *configs, rebuild_adapters=False):
+    """Configure UCR processors and discard other pillow processors"""
     configs_by_domain = {}
     for config in configs:
         configs_by_domain.setdefault(config.domain, []).append(config)
 
     _SHARED_ADAPTER_CACHES.clear()
-    for proc in pillow.processors:
+    for proc in list(pillow.processors):
         if hasattr(proc, 'table_manager'):
```

Comment:
nit: With this change, is there any more obvious config difference to check? Having an attribute "table_manager" works, but just feels like there isn't anything about that that is uniquely specific to UCR, and if there was a way to just say "if proc.__class__ in [...]" or something like that, it might be a fit a bit better.